src/main/java/seedu/address/model/student/admin/ClassTime.java

```diff
@@ -18,7 +18,7 @@
 
     public static final String MESSAGE_CONSTRAINTS =
             "Class Time should follow the following format: {int: day_of_week} {int: start_time}-{int: end_time}";
-    public static final String TIME_CONSTRAINTS = "End time should always be after Start time";
+    public static final String TIME_CONSTRAINTS = "Invalid Class Time! \nEnd time should always be after Start time";
```

Comment:
Do you think
`TIME_CONSTRAINTS = "Classes should not end before they start."

Reply:
Yeap.